client/components/result.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import Card from 'react-bootstrap/Card';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+
+export default function result(props) {
+  const { name, distance, categories, display_phone: phone } = props.result;
+  const { address1, city, state, zip_code: zipCode } = props.result.location;
+  const miles = distance * 0.000621371192;
```

Comment:
This is a "magic number". Please assign it to a `CONSTANT_VARIABLE` at the top of your module with a descriptive name.

https://en.wikipedia.org/wiki/Magic_number_(programming)